src/screens/login/OnboardingScreen.js

```diff
@@ -154,6 +203,16 @@ const OnboardingSteps = ({ setClient, route, navigation }) => {
     )
   }
 
+  if (state.step === MAGIC_LINK_STEP) {
+    hideSplashScreen()
+    const { fqdn } = state.onboardingData
+    return (
+      <View>
+        <Text>Magic Onboarding on {fqdn}</Text>
+      </View>
+    )
+  }
```

Comment:
is it really wanted this screen?

Reply:
No it is not wanted. I forgot to mention that, but this part is temporary. I didn't implement the final version yet as this is not in the critical path and it is discreet (displayed half a second before we get the authorization IAB or the home navigation)